cts-java/src/test/java/org/ga4gh/cts/api/variants/VariantsPagingIT.java

```diff
@@ -0,0 +1,152 @@
+package org.ga4gh.cts.api.variants;
+
+import org.apache.avro.AvroRemoteException;
+import org.ga4gh.ctk.CtkLogs;
+import org.ga4gh.ctk.transport.URLMAPPING;
+import org.ga4gh.ctk.transport.protocols.Client;
+import org.ga4gh.cts.api.TestData;
+import org.ga4gh.cts.api.Utils;
+import org.ga4gh.methods.GAException;
+import org.ga4gh.methods.SearchVariantsRequest;
+import org.ga4gh.methods.SearchVariantsResponse;
+import org.ga4gh.models.Variant;
+import org.ga4gh.models.VariantSet;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Test the <tt>/variants/search</tt> paging behavior.
+ *
+ * @author Herb Jellinek
+ */
+@Category(VariantsTests.class)
+public class VariantsPagingIT implements CtkLogs {
+
+    private static Client client = new Client(URLMAPPING.getInstance());
+
+    /**
+     * Check that we can page 1 by 1 through the variants we receive from
+     * {@link org.ga4gh.ctk.transport.protocols.Client.Variants#searchVariants(SearchVariantsRequest)}.
+     *
+     * @throws AvroRemoteException if there's a communication problem or server exception ({@link GAException})
+     */
+    @Test
+    public void checkPagingOneByOneThroughCalls() throws AvroRemoteException {
+        final long start = 50;
+        final long end = 100;
+
+        final String variantSetId = Utils.getVariantSetId(client);
+        final List<Variant> listOfVariants = Utils.getAllVariantsInRange(client, variantSetId, start, end);
+
+        // we will remove Variants from this Set and assert at the end that we have zero
+        final Set<Variant> setOfVariants = new HashSet<>(listOfVariants);
+        assertThat(listOfVariants).hasSize(setOfVariants.size());
+
+        // page through the variants using the same query parameters
+        String pageToken = null;
+        for (Variant ignored : listOfVariants) {
```

Comment:
This test is written as if start/end were an exact match query - it's actually an [overlap query](https://github.com/ga4gh/schemas/blob/master/src/main/resources/avro/variantmethods.avdl#L84) though.

Either this test doesn't work, or it happens to work because no two variants happen to overlap in the test data. If it's the latter, I don't think it makes sense to depend on that.